lib/snapshot/BUILD.gn

```diff
@@ -62,9 +67,13 @@ action("generate_snapshot_bin") {
     "--snapshot_kind",
     "core",
     "--vm_output_bin",
-    rebase_path(vm_isolate_snapshot, root_build_dir),
+    rebase_path(vm_snapshot_data, root_build_dir),
```

Comment:
Do you need to update create_snapshot_bin.py to pass along --load_compilation_trace to gen_snapshot?